app/views/devise/registrations/new.html.erb

```diff
@@ -19,6 +20,10 @@
     <%= f.input :password_confirmation,
                 required: true,
                 input_html: { autocomplete: "new-password" } %>
+    <%= f.input :photo,
+                label: "Upload your picture (optional)",
+                required: false,
+                input_html: { autocomplete: "photo" } %>
```

Comment:
I think we should remove autocomplete for photo

Reply:
Change applied!